java-frontend/src/test/java/org/sonar/java/UCFGJavaVisitorTest.java

```diff
@@ -325,21 +345,26 @@ private UCFG assertCodeToUCfg(String source, UCFG expectedUCFG, boolean testLoca
     } catch (IOException e) {
       e.printStackTrace();
     }
-    assertThat(actualUCFG.methodId()).isEqualTo(expectedUCFG.methodId());
-    assertThat(actualUCFG.basicBlocks()).isEqualTo(expectedUCFG.basicBlocks());
-    assertThat(actualUCFG.basicBlocks().values().stream().flatMap(b->b.calls().stream()))
-      .containsExactlyElementsOf(expectedUCFG.basicBlocks().values().stream().flatMap(b->b.calls().stream()).collect(Collectors.toList()));
-    assertThat(actualUCFG.basicBlocks().values().stream().map(BasicBlock::terminator))
-      .containsExactlyElementsOf(expectedUCFG.basicBlocks().values().stream().map(BasicBlock::terminator).collect(Collectors.toList()));
-    assertThat(actualUCFG.entryBlocks()).isEqualTo(expectedUCFG.entryBlocks());
-    assertThat(toLocationStream(actualUCFG).noneMatch(l->l == UCFGBuilder.LOC)).isTrue();
-    if(testLocations) {
-      Stream<LocationInFile> locStream = toLocationStream(actualUCFG);
-      assertThat(locStream).containsExactlyElementsOf(toLocationStream(expectedUCFG).collect(Collectors.toList()));
-    }
     return actualUCFG;
   }
 
+  @Test
+  public void method_with_unknown_symbol_should_not_produce_ucfg() {
+    CompilationUnitTree cut = getCompilationUnitTreeWithSemantics("class A {void foo(UnkownType t) {return t;}}");
```

Comment:
move this line next to the `UCFGJavaVisitor.visitCompilationUnit(cut);` line, in order to have both test cases close to each other. Difference is not explicit at first glance. Maybe add a comment on top of each case.